src/configure

```diff
@@ -896,6 +945,41 @@ fi
 # Most of the OS-specific steps below will append to kaldi.mk
 echo "Doing OS specific configurations ..."
 
+if $android ; then
+  OPENFSTLIBS="$FSTROOT/lib/libfst.a"
+  echo "OPENFSTLIBS = $OPENFSTLIBS" >> kaldi.mk
+
+  if [ -z $TOOLCHAININC ] ;  then
+    failure "--toolchain-includes must be specified for android builds"
+  fi
+
+  if [ -z $HOST ] ; then
```

Comment:
Do you mean we are building on a linux x86_64 system, but have set --target-arch=arm?

I hadn't thought to try that, but it seems to configure "successfully" but the build immediately fails when you try and compile because it passes arm specific flags to an x86_64 compiler.

I will update the configure script to fail is --target-arch is set to a non-native value but --host is not set.